gensim/models/wrappers/fasttext.py

```diff
@@ -123,11 +123,8 @@ def __contains__(self, word):
         if word in self.vocab:
             return True
         else:
-            word_ngrams = set(FastText.compute_ngrams(word, self.min_n, self.max_n))
-            if len(word_ngrams & set(self.ngrams.keys())):
-                return True
-            else:
-                return False
+            word_ngrams = FastText.compute_ngrams(word, self.min_n, self.max_n)
+            return any(ng in self.ngrams for ng in word_ngrams)
```

Comment:
Thanks! The original was a bizarre construct indeed.

How often can we expect `compute_ngrams` to contain duplicates? If it's a non-trivial number, iterating over `set(word_ngrams)` better.